modin/tests/pandas/dataframe/test_join_sort.py

```diff
@@ -1012,3 +1012,45 @@ def test_compare(align_axis, keep_shape, keep_equal):
     modin_result = modin_series2.compare(modin_series1, **kwargs)
     pandas_result = pandas_series2.compare(pandas_series1, **kwargs)
     assert to_pandas(modin_result).equals(pandas_result)
+
+
+@pytest.mark.parametrize(
+    "params",
+    [
+        {"ascending": True},
+        {"normalize": True},
+        pytest.param(
+            {"sort": False},
+            marks=pytest.mark.xfail(
+                reason="Known issue with sort=False in `groupby()` \
+                    (https://github.com/modin-project/modin/issues/3571)",
+                strict=False,
```

Comment:
```suggestion
                strict=Engine.get() == "Ray",
```

If it's a Ray-only bug, then it should be expected to fail only on Ray. If the failure pops up on any other engine then it's a regression.